test/CadlRanchProjects/encode/duration/src/Generated/Header.cs

```diff
@@ -402,7 +402,7 @@ internal HttpMessage CreateIso8601ArrayRequest(IEnumerable<TimeSpan> duration, R
             uri.AppendPath("/encode/duration/header/iso8601-array", false);
             uri.AppendQuery("api-version", _apiVersion, true);
             request.Uri = uri;
-            request.Headers.AddDelimited("duration", duration, ",");
+            request.Headers.AddDelimited("duration", duration, ",", "P");
```

Comment:
Could you help to explain this change?  Is it expected?

Reply:
It is a duration array, we need format each array item to right format, so we add format as the function parameter.